doc: Add the guide for integrating with IFTTT (#3092)
TheThingsNetwork/lorawan-stack: doc/content/integrations/webhooks/cloud-integrations/ifttt/tts-setup/_index.md

---
title: "Creating a Webhook"
description: ""
weight: 2
---

Follow this section to create a Webhook integration with the IFTTT applet.

<!--more-->

>Note: this section follows the [HTTP Webhooks]({{< ref "/integrations/webhooks" >}}) guide. 

Fill in the **Webhook ID** field. 

As a **Webhook format**, choose **JSON**.

Paste the previously copied URL from IFTTT in the **Base URL** field.

Tick the box next to the message types which you want to enable this webhook for and select **Add webhook**.

{{< figure src="creating-a-webhook.png" alt="IFTTT webhook" >}}

> If you have followed this guide literally, you will receive an SMS message once the join request from your device is accepted.

Reviewer: ```suggestion
Once you have completed the integration, you will receive an sms whenever a join request from your device is accepted.
```
